site/content/en/v1.0.0/user/quickstart.md

```diff
@@ -47,10 +47,16 @@ consideration when debugging.
 
 ## Testing the Configuration
 
+{{< tabs name="tabs_test_the_configuration" >}}
+{{% tab name="Test Configuration" %}}
+
 Get the name of the Envoy service created the by the example Gateway:
 
 ```shell
-export ENVOY_SERVICE=$(kubectl get svc -n envoy-gateway-system --selector=gateway.envoyproxy.io/owning-gateway-namespace=default,gateway.envoyproxy.io/owning-gateway-name=eg -o jsonpath='{.items[0].metadata.name}')
+export ENVOY_SERVICE= \ 
+    $(kubectl get svc -n envoy-gateway-system \
+    --selector=gateway.envoyproxy.io/owning-gateway-namespace=default,gateway.envoyproxy.io/owning-gateway-name=eg \
+    -o jsonpath='{.items[0].metadata.name}')
```

Comment:
why need this?

Reply:
Because the previous command was too long to display well, I added line breaks

Reply:
can you share the diff with a image?

Reply:
before: 
![image](https://github.com/envoyproxy/gateway/assets/77964041/d684472e-3e86-41ff-b23a-1f8741856873)

after: ( is also what it looks like when a tab is added)
![image](https://github.com/envoyproxy/gateway/assets/77964041/f67fd169-1b5c-4aec-8105-70f4ff338442)